lib/pbench/server/database/database.py

```diff
@@ -56,15 +65,26 @@ def wait_for_database(db_uri: str, timeout: int):
                 time.sleep(1)
 
     @staticmethod
-    def create_if_missing(db_uri, logger):
-        """Create the database if it doesn't exist."""
+    def create_if_missing(db_uri: str, logger: Logger):
+        """Create the database if it doesn't exist.
+
+        Args:
+            db_uri: The server's DB URI
+            logger: A Python logger object
+        """
         if not database_exists(db_uri):
             logger.info("Database {} doesn't exist", db_uri)
             create_database(db_uri)
             logger.info("Created database {}", db_uri)
```

Comment:
This will be replaced by the Flask-Alembic migration when we get around to doing that. For now, I did nothing but improve the docstring and add type hints!